### PR TITLE
Move away from word "cloud"

### PR DESCRIPTION
**Pull request by Kirastina**

### CODE REVIEW

File: site/content/english/integration.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ og_image: og/og-skribble-identification.jpg
 {{< content class="no-bottom-padding" >}}
 {{% heading %}}
 # Skribble Integration
-With Skribble, you choose a cloud platform that combines
-technological security with simplicity.
+With Skribble, you choose an online platform that combines technological security with simplicity.
```
Comment (Kirastina) [inline site/content/english/integration.md]:
```suggestion
With Skribble, you choose an online platform that combines
technological security with simplicity.
```

File: site/content/french/integration.md [hunk 2]
```diff
@@ -12,8 +12,7 @@ og_image: og/og-skribble-identification.jpg
 {{< content >}}
 {{% heading %}}
 # Intégration de Skribble
-Avec Skribble, vous choisissez une plate-forme de cloud
-qui allie sécurité technologique et simplicité.
+Avec Skribble, vous choisissez une plate-forme en ligne qui allie sécurité technologique et simplicité.
```
Comment (Kirastina) [inline site/content/french/integration.md]:
```suggestion
Avec Skribble, vous choisissez une plate-forme en ligne
qui allie sécurité technologique et simplicité.
```

File: site/content/german/integration.md [hunk 3]
```diff
@@ -12,8 +12,7 @@ og_image: og/og-skribble-identification.jpg
 {{< content >}}
 {{% heading %}}
 # Skribble Integration
-Mit Skribble wählst du eine Cloud-Plattform, 
-die technologische Sicherheit und Einfachheit vereint.
+Mit Skribble wählst du eine Online-Plattform, die technologische Sicherheit und Einfachheit vereint.
```
Comment (Kirastina) [inline site/content/german/integration.md]:
```suggestion
Mit Skribble wählst du eine Online-Plattform, die
technologische Sicherheit und Einfachheit vereint. 
```

**Comment by betatim** [comment]
@fursinger is it Ok if we merge this and add it to the list of PRs in the Asana ticket?

File: site/content/german/integration.md [hunk 4]
```diff
@@ -12,8 +12,8 @@ og_image: og/og-skribble-identification.jpg
 {{< content text=center top=16 bottom=8 width=narrow >}}
 {{% markdown %}}
 # Skribble Integration
-Mit Skribble wählst du eine Cloud-Plattform,
-die technologische Sicherheit und Einfachheit vereint.
+Mit Skribble wählst du eine Online-Plattform, die
+technologische Sicherheit und Einfachheit vereint.
```
Comment (betatim) [inline site/content/german/integration.md]:
```suggestion
technologische Sicherheit und Einfachheit vereint.
```

**Pull request merged**